### ISSUE TITLE
Guide for RT

### ISSUE THREAD

**Issue by nicostuhlfauth** [Uberspace/lab#329]
Please add a guide for RT on Uberspace 7.

**Comment by netbuk** [comment]
You mean this one > https://bestpractical.com/request-tracker ?

**Comment by nicostuhlfauth** [comment]
Yes, this one. General installation and how-to configure PGP would be really interesting for me. Maybe someone has already installed it?!
